public/components/notebooks/components/note_table.tsx

```diff
@@ -56,33 +56,43 @@ interface NoteTableProps {
   deleteNotebook: (noteList: string[], toastMessage?: string) => void;
   parentBreadcrumb: ChromeBreadcrumb;
   setBreadcrumbs: (newBreadcrumbs: ChromeBreadcrumb[]) => void;
-  setToast: (title: string, color?: string, text?: string) => void;
+  // setToast: (title: string, color?: string, text?: string) => void;
 }
 
-export function NoteTable(props: NoteTableProps) {
+export function NoteTable({
+  loading,
+  fetchNotebooks,
+  addSampleNotebooks,
+  notebooks,
+  createNotebook,
+  renameNotebook,
+  cloneNotebook,
+  deleteNotebook,
+  parentBreadcrumb,
+  setBreadcrumbs,
+}: NoteTableProps) {
```

Comment:
curious does lint require this change?

Reply:
```
.../public/components/notebooks/components/note_table.tsx
  78:6  warning  React Hook useEffect has a missing dependency: 'props'. Either include it or remove the dependency array. However, 'props' will change when *any* prop changes, so the preferred fix is to destructure the 'props' object outside of the useEffect call and refer to those specific props inside useEffect  react-hooks/exhaustive-deps
```
Removing the useEffect dependency array makes it infinite-loop and blocks render.
The most-right thing to do is move the props destructuring up into the declaration, since most of the props were destructured in the first lines of constructor anyway.